CONTRIBUTING.md

```diff
@@ -52,6 +52,27 @@ Use the following commands:
 
 See our GitHub Actions [`tests.yml`](.github/workflows/tests.yml) for further reference.
 
+## Using `pytest-recording` and VCR cassettes
+
+We use the [`pytest-recording`](https://github.com/kiwicom/pytest-recording) plugin
+to create VCR cassettes to cache HTTP requests,
+making our unit tests more deterministic.
+
+To record a new VCR cassette:
+
+```bash
+uv run pytest tests/desired_test_module.py
```

Comment:
I thought this wasn't true because record_mode is `none` -- right? Don't we need to override it to create a new cassette?

Reply:
Alright, adjusted, and removed the `conftest.py` default that was blocking `--record-mode` CLI flag